app/assets/javascripts/discourse/app/components/add-category-tag-classes.js

```diff
@@ -15,36 +15,43 @@ export default Component.extend({
       return;
     }
     const slug = this.get("category.fullSlug");
-    const tags = this.tags;
 
     this._removeClass();
 
-    let classes = [];
+    const classes = [];
+
     if (slug) {
       classes.push("category");
       classes.push(`category-${slug}`);
     }
-    if (tags) {
-      tags.forEach((t) => classes.push(`tag-${t}`));
-    }
-    if (classes.length > 0) {
-      $("body").addClass(classes.join(" "));
-    }
+
+    this.tags?.forEach((t) => classes.push(`tag-${t}`));
+
+    document.documentElement.classList.add(...classes);
```

Comment:
Previously it was applying the classes on `<body>` not `<html>`. Is this change intended?

Reply:
😱 

Don't know why I thought it was HTML, thanks.